### DISCUSSION TITLE
[mini-files] background color

### DISCUSSION THREAD

**Discussion by LintaoAmons** [nvim-mini/mini.nvim#1178] [Q&A] [answered]
Is there a way to change the background colour of the mini-files buffer? 

I found it quite hard to differentiate the buffers.

It took a while until I found the buffer already being there after I triggered the keybinding.

**Comment by echasnovski** [comment, marked as answer]
Yes, there is a [`MiniFilesNormal`](https://github.com/echasnovski/mini.nvim/blob/42744fd81136fdd7e01ce81fb99fc6b5e9a17f0f/doc/mini-files.txt#L94) highlight group. You can modify it as any other highlight group. For example, with the following Lua call: `vim.api.nvim_set_hl(0, 'MiniFilesNormal', { bg = '#111111' })`.

But more so, this sounds like the color scheme you are using doesn't show floating windows with the distinctive background color. I'd recommend adjusting `NormalFloat` highlight group.
